js/files_test.ts

```diff
@@ -153,3 +153,53 @@ testPerm({ read: true, write: true }, async function openModeWriteRead() {
 
```

Comment:
@kevinkassimo can you fix this test case and remove TODO?

Reply:
Done.